### PR TITLE
BlanchardLj's final-project

### PR DESCRIPTION
**Pull request by BlanchardLj**
I completed the homework assigned for the course. I completed the learning summary and the questions which are about Git.

### CODE REVIEW

**Comment by BlanchardLj** [comment]
Need review. @irvingskr

**Comment by irvingskr** [comment]
> Need review. @irvingskr

Please write something before call me to review.

**Comment by BlanchardLj** [comment]
I completed the homework assigned for the course. I completed the learning summary and the questions which are about Git.Please check it.

**Comment by BlanchardLj** [comment]
I completed the homework assigned for the course. I completed the learning summary and the questions which are about Git.I have completed the 2022-finalproject, and please check it. @irvingskr

File: 2022-finalproject/BlanchardLj/git.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+1. 若你已经修改了部分文件、并且将其中的一部分加入了暂存区，应该如何回退这些修改，恢复到修改前最后一次提交的状态？给出至少两种不同的方式
+   
+   ![](./1-1.PNG)
+
+   ![](./1-2.PNG)
+
+   ![](./1-3.PNG)
```
Comment (irvingskr) [inline 2022-finalproject/BlanchardLj/git.md:7]: You just modified the files in the index, but the task required you to return the working area to the condition of the last commit,.

**Review by irvingskr — CHANGES_REQUESTED** [review]

File: 2022-finalproject/BlanchardLj/git.md (hunk 1 above)
Comment (BlanchardLj) [reply 2022-finalproject/BlanchardLj/git.md:7]: OK, I have changed it. Please check it again. @irvingskr

**Review by irvingskr — APPROVED** [review]

**Comment by BlanchardLj** [comment]
Sorry, my phone is broken.I just saw the request. Please review it. @irvingskr

**Comment by BlanchardLj** [comment]
view it on GitHub
&nbsp;


rolling&nbsp;star
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月11日(星期五) 凌晨1:38
收件人: ***@***.***&gt;; 
抄送: "rolling ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [gdut-dynamic-x/missing-semester] BlanchardLj&#39;s final-project (PR #73)





 
Merged #73 into master.

**Pull request merged**